microsoft/PowerToys
# StyleCop and warnings fix

## Summary of the Pull Request

**What is this about:**
StyleCop instelled and warnings fix.

**What is include in the PR:** 
Installed StyleCop on missing projects and fixed build warnings.

**How does someone test / validate:** 
- PT Run VS Code Workspaces plugin working as expected
- PT Run Unit Converter plugin working as expected

## Quality Checklist

- [x] **Linked issue:** #12370 (StyleCop and warnings only)

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
